### ISSUE TITLE
Add plot_maneuver

### ISSUE THREAD

**Issue by MLopez-Ibanez** [poliastro/poliastro#1349]
💡 **Feature request**

Currently sample() accepts a min and max anomaly. However, it is not always clear how to set these values. For example, in the [Going to Jupiter example](https://docs.poliastro.space/en/stable/examples/Going%20to%20Jupiter%20with%20Python%20using%20Jupyter%20and%20poliastro.html) we have code like:

```python
plotter.plot_trajectory(
    ic3.sample(min_anomaly=10 * u.deg, max_anomaly=180 * u.deg),
    label="Inner cruise 3",
    color="C3",
)
```

It is not clear why one should set 10 and 180 here. 

In our particular case, we know the start and end time of the maneuver.

Another alternative could be to provide a `plot_maneuver` function that automatically takes care of plotting the whole maneuver but just that, ideally plotting also any intermediate impulses. This way, one could do:

```python
man_flyby = Maneuver.lambert(ic1_end, ss_efly)
plotter.plot_maneuver(man_flyby, label="Inner cruise 1", color="C1")
```

**Comment by MLopez-Ibanez** [comment]
@jorgepiloto Do you know how this could be implemented?

**Comment by astrojuanlu** [comment]
Fixed in #1359
